test/passes/fannkuch3_manyopts.bin.txt

```diff
@@ -2265,622 +2368,760 @@ Abbrev table for offset: 0x00000000
-                DW_AT_type [DW_FORM_ref4]	(cu + 0x0045 => {0x00000045} "int")
+                DW_AT_type [DW_FORM_ref4]	(cu + 0x0059 => {0x00000059} "int")
                 DW_AT_external [DW_FORM_flag_present]	(true)
 
-0x000000c9:     DW_TAG_formal_parameter [16]  
-                  DW_AT_name [DW_FORM_strp]	( .debug_str[0x000001bf] = "_arg")
-                  DW_AT_decl_file [DW_FORM_data1]	("/usr/local/google/home/azakai/Dev/emscripten/fannkuch.cpp")
+0x0000009e:     DW_TAG_formal_parameter [11]  
+                  DW_AT_name [DW_FORM_strp]	( .debug_str[0x00000191] = "_arg")
+                  DW_AT_decl_file [DW_FORM_data1]	("/usr/local/google/home/azakai/Dev/emscripten/tests/fannkuch.cpp")
                   DW_AT_decl_line [DW_FORM_data1]	(26)
-                  DW_AT_type [DW_FORM_ref4]	(cu + 0x0033 => {0x00000033} "*")
+                  DW_AT_type [DW_FORM_ref4]	(cu + 0x01a7 => {0x000001a7} "*")
 
-0x000000d4:     DW_TAG_variable [17]  
-                  DW_AT_name [DW_FORM_strp]	( .debug_str[0x00000161] = "args")
-                  DW_AT_decl_file [DW_FORM_data1]	("/usr/local/google/home/azakai/Dev/emscripten/fannkuch.cpp")
+0x000000a9:     DW_TAG_variable [12]  
+                  DW_AT_name [DW_FORM_strp]	( .debug_str[0x00000133] = "args")
+                  DW_AT_decl_file [DW_FORM_data1]	("/usr/local/google/home/azakai/Dev/emscripten/tests/fannkuch.cpp")
                   DW_AT_decl_line [DW_FORM_data1]	(28)
-                  DW_AT_type [DW_FORM_ref4]	(cu + 0x005d => {0x0000005d} "worker_args*")
+                  DW_AT_type [DW_FORM_ref4]	(cu + 0x0026 => {0x00000026} "worker_args*")
 
-0x000000df:     DW_TAG_variable [18]  
+0x000000b4:     DW_TAG_variable [13]  
                   DW_AT_location [DW_FORM_sec_offset]	(0x00000000: 
-                     [0xffffffff,  0x00000003): 
+                     [0xffffffff,  0x00000006): 
                      [0x00000000,  0x0000003c): DW_OP_consts +0, DW_OP_stack_value)
-                  DW_AT_name [DW_FORM_strp]	( .debug_str[0x00000174] = "maxflips")
-                  DW_AT_decl_file [DW_FORM_data1]	("/usr/local/google/home/azakai/Dev/emscripten/fannkuch.cpp")
+                  DW_AT_name [DW_FORM_strp]	( .debug_str[0x0000014c] = "maxflips")
+                  DW_AT_decl_file [DW_FORM_data1]	("/usr/local/google/home/azakai/Dev/emscripten/tests/fannkuch.cpp")
                   DW_AT_decl_line [DW_FORM_data1]	(30)
-                  DW_AT_type [DW_FORM_ref4]	(cu + 0x0045 => {0x00000045} "int")
+                  DW_AT_type [DW_FORM_ref4]	(cu + 0x0059 => {0x00000059} "int")
 
-0x000000ee:     DW_TAG_variable [18]  
+0x000000c3:     DW_TAG_variable [13]  
                   DW_AT_location [DW_FORM_sec_offset]	(0x0000001d: 
-                     [0xffffffff,  0x00000003): 
+                     [0xffffffff,  0x00000006): 
                      [0x00000007,  0x0000003c): DW_OP_consts +0, DW_OP_stack_value
+                     [0x00000052,  0x00000057): <decoding error> 00 01 9f
```

Comment:
this is also expected, as it comes from a llvm.dbg.value intrinsic and should also encode a wasm local.